INSTALL.md

```diff
@@ -1,5 +1,84 @@
-Building Dogecoin
-================
+# Building Dogecoin Core
 
-See doc/build-*.md for instructions on building the various
-elements of the Dogecoin Core reference implementation of Dogecoin.
+Development is ongoing, and the development team, as well as other volunteers,
+can freely work in their own trees and submit pull requests when features or
+bug fixes are ready.
+
+### Wow plz make dogecoind/dogecoin-cli/dogecoin-qt
+
+  The following are developer notes on how to build Dogecoin on your native platform. They are not complete guides, but include notes on the necessary libraries, compile flags, etc.
+
+  - [OSX Build Notes](doc/build-osx.md)
+  - [Unix Build Notes](doc/build-unix.md)
+  - [Windows Build Notes](doc/build-windows.md)
+
+### Such ports
+
+- RPC 22555
+- P2P 22556
+
+#### Version strategy
+Version numbers are following ```major.minor.patch``` semantics.
+
+#### Branches
+There are 3 types of branches in this repository:
+
+- **master:** Stable, contains the latest version of the latest *major.minor* release.
+- **maintenance:** Stable, contains the latest version of previous releases, which are still under active maintenance. Format: ```<version>-maint```
+- **development:** Unstable, contains new code for planned releases. Format: ```<version>-dev```
+
+*Master and maintenance branches are exclusively mutable by release. Planned*
+*releases will always have a development branch and pull requests should be*
+*submitted against those. Maintenance branches are there for **bug fixes only,***
+*please submit new features against the development branch with the highest version.*
+
+#### Contributions ✍️
+
+Developers are strongly encouraged to write [unit tests](src/test/README.md) for new code, and to
+submit new unit tests for old code. Unit tests can be compiled and run
+(assuming they weren't disabled in configure) with: `make check`. Further details on running
+and extending unit tests can be found in [/src/test/README.md](/src/test/README.md).
+
+There are also [regression and integration tests](/qa) of the RPC interface, written
+in Python, that are run automatically on the build server.
+These tests can be run (if the [test dependencies](/qa) are installed) with: `qa/pull-tester/rpc-tests.py`
+
+Changes should be tested by somebody other than the developer who wrote the
+code. This is especially important for large or high-risk changes. It is useful
+to add a test plan to the pull request description if testing the changes is
+not straightforward.
+
+
+## Development tips and tricks
+
+**compiling for debugging**
+
+Run `configure` with the `--enable-debug` option, then `make`. Or run `configure` with
+`CXXFLAGS="-g -ggdb -O0"` or whatever debug flags you need.
```

Comment:
This is a little unclear; could it be rewritten to make the commands a little more obvious?

    `./configure --enable-debug`
    
    ` ./configure CXXFLAGS="-g -ggdb -O0"`
    
This is more consistent with the lock debugging example later.

Reply:
Actually I just moved all technical information from the README and put it inside INSTALL.md (perhaps too much). I didn't worked on the install process for now.